test/observe_test.dart

```diff
@@ -106,6 +108,7 @@ void _observeTests(createModel(x)) {
 
     subs.add(t.changes.listen((n) {}));
     expect(t.hasObservers, true);
+    Observable.dirtyCheck();
```

Comment:
I'm surprised we need all these - is `test` less asynchronous than unittest was?